mkn.yaml

```
@@ -0,0 +1,217 @@
+
+- name: base
+  parent: ${mkn.base}
+
+
+- name: core
+  parent: base
+  inc: src subprojects subprojects/phlop/inc
+  src: src/core subprojects/phlop/src/phlop/timing/threaded_scope_timer.cpp
+  out: phare_core
+  sub:
+      cppdict#master&subprojects/cppdict(https://github.com/LaboratoryOfPlasmaPhysics/cppdict)
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)